r/R/dplyr.R

```diff
@@ -110,6 +110,8 @@ make_field_refs <- function(field_names) {
 #' @export
 print.arrow_dplyr_query <- function(x, ...) {
   schm <- x$.data$schema
+  schm[["__filename"]] <- string()
```

Comment:
So this worked? Great! Do you want to add a comment here explaining what's going on, or encapsulate this augmented column business into a well named function?

Reply:
Yeah, was very pleased that it was this simple in the end! Comment added.